client/src/pages/lessons/edit.tsx

```diff
@@ -894,194 +651,299 @@ const LessonEdit = (props: {
+                setLesson(
+                  {
+                    ...(lessonUnderEdit.lesson || newLesson),
+                    conclusion: conclusions,
+                  },
+                  true
+                )
+              }
+            />
+          </form>
+          <div
+            style={{
+              display: "flex",
+              alignItems: "center",
+              padding: 5,
+              cursor: "pointer",
+            }}
+            onClick={() =>
+              setIsShowingAdvancedFeatures(!isShowingAdvancedFeatures)
             }
-          }}
-        >
-          Train
-        </Button>
-        <Button
-          data-cy="launch-button"
-          variant="contained"
-          endIcon={<LaunchIcon />}
-          color="primary"
-          size="large"
-          disabled={!lessonId || !isLessonValid()}
-          onClick={handleLaunch}
-        >
-          Launch
-        </Button>
-        {lessonUnderEdit.dirty ? (
-          <Button
-            data-cy="save-button"
-            variant="contained"
-            startIcon={<SaveIcon />}
-            color="primary"
-            size="large"
-            onClick={() => handleSavePopUp(true)}
-            disabled={!isLessonValid()}
-          >
-            Save
-          </Button>
-        ) : null}
-        {isDownloadable ? (
-          <Button
-            data-cy="download-button"
-            variant="contained"
-            startIcon={<Download />}
-            color="primary"
-            size="large"
-            onClick={download}
-            disabled={isDownloading}
-          >
-            Download
-          </Button>
-        ) : null}
-      </div>
-      <Dialog open={Boolean(trainingMessage)} onClose={dismissTrainingMessage}>
-        <DialogTitle>{trainingMessage}</DialogTitle>
-      </Dialog>
-      <Dialog open={Boolean(downloadMessage)} onClose={dismissDownloadMessage}>
-        <DialogTitle>{downloadMessage}</DialogTitle>
-      </Dialog>
-      <Dialog open={savePopUp} onClose={() => handleSavePopUp(false)}>
-        <DialogTitle>Save</DialogTitle>
-        <DialogActions>
-          <Button data-cy="save-exit" onClick={handleSaveExit} color="primary">
-            Exit
-          </Button>
-          <Button
-            data-cy="save-continue"
-            onClick={handleSaveContinue}
-            color="primary"
-            variant="contained"
           >
-            Continue
-          </Button>
-        </DialogActions>
-      </Dialog>
-      <ToastContainer />
-    </div>
+            {isShowingAdvancedFeatures ? <ArrowDropDown /> : <ArrowRight />}
+            <Typography variant="body2" data-cy="advanced-features">
+              {isShowingAdvancedFeatures
+                ? "Hide Advanced Features"
+                : "Show Advanced Features"}
+            </Typography>
+          </div>
+
+          <div style={isShowingAdvancedFeatures ? {} : { display: "none" }}>
```

Comment:
looks like this whole div for displaying "advanced features" could be moved into its own component (for better readability/maintainability/etc.)